integration-tests/src/main/java/io/druid/testing/clients/QueryResourceTestClient.java

```diff
@@ -93,6 +100,7 @@ private String getBrokerURL()
       );
     }
     catch (Exception e) {
+      log.error("exception: " + e.getMessage());
```

Comment:
if the exception is propagated then why is the log needed, whoever handles the exception will log ?however, if we do log, then we should probably do `log.error(e, "exception")`